src/common/layout/PageLayout.tsx

```diff
@@ -10,11 +11,23 @@ import styles from './PageLayout.module.css';
 
 type Props = {
   children: ReactNode;
+  title?: string;
   background: 'youth' | 'adult';
 };
 
-function PageLayout(props: Props) {
+function PageLayout({ children, title = 'appName', background }: Props) {
   const { t, i18n } = useTranslation();
+  const { trackPageView } = useMatomo();
+
+  const pageTitle =
+    title !== 'appName' ? `${t(title)} - ${t('appName')}` : t('appName');
+
+  useEffect(() => {
```

Comment:
This effect does not have a deps array, so it's run on every render. By adding an empty deps array it is ran on mount. By adding `pageTitle` and `href` it'd be run when those two values change.

I don't know much about matomo so I don't know if this is an issue.

Reply:
I somehow managed to ignore run time warning by IDE threw about this. Dependency array is now added.